src/main/java/eu/elixir/ega/ebi/egafuse/EgaFuse.java

```diff
@@ -705,4 +711,9 @@ public int open(String path, FuseFileInfo fi) {
 
         return 0;
     }
+
+    public static int getConnection() {
+        return connection;
+    }
```

Comment:
Maybe we can add Lombok library so we don't need to manually write getters and setters?

Reply:
As a general rule we don't use Lombok. Getters can be autogenerated on a case by case basis and we prefer to maintain control of what do we want to be accesible.